core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -1157,6 +1181,16 @@ public SELF withCreateContainerCmdModifier(Consumer<CreateContainerCmd> modifier
         return self();
     }
 
+    /**
+     * Size of /dev/shm
+     * @param bytes The number of megabytes to assign the shared memory. If null, it will apply the Docker default which is 64 KB.
```

Comment:
```suggestion
     * @param bytes The number of megabytes to assign the shared memory. If null, it will apply the Docker default which is 64 MB.
```

Isn't it MB?

Reply:
Darn it! I went off the Oracle docs but you are write.

http://www.tin.org/bin/man.cgi?section=8&topic=dockerd

```
    --default-shm-size=64MiB
         Set the daemon-wide default shm size for containers. Default is
       64MiB.
```